test1-prod.js

```diff
@@ -7,4 +7,6 @@ if(fooProd  === "bar") {
   console.log( 'sha from event' )
   
   console.log( 'sha checking event name for real with logs' )
+  
+  console.log( 'only final notification, this issue did not come from GH' )
```

Comment:
Codacy has a fix for the issue: [Replace `·'only·final·notification,·this·issue·did·not·come·from·GH'·)` with `"only·final·notification,·this·issue·did·not·come·from·GH");`](https://app.dev.codacy.org/gh/troubleshoot-codacy/eslint-test-prod/pullRequest?prid=33432)
```suggestion
  console.log("only final notification, this issue did not come from GH");
```